### PR TITLE
Inline doc for DataFrame.assign

### PR DESCRIPTION
**Pull request by shril**

### CODE REVIEW

**Comment by shril** [comment]
Hi @rxin, I need your help here.

I am getting 2 different outputs for Python 3.5 and Python 3.6 for the same function.

```
df = ks.DataFrame({'temp_c': [17.0, 25.0]}, index=['Portland', 'Berkeley'])
```

In Python 3.6 -
```
>>> df.assign(temp_k=df['temp_c'] + 273.15,
...           temp_f=df['temp_c'] * 9 / 5 + 32)

          temp_c  temp_k  temp_f
Portland    17.0  290.15    62.6
Berkeley    25.0  298.15    77.0
```

In Python 3.5 - 
```
>>> df.assign(temp_k=df['temp_c'] + 273.15,
...           temp_f=df['temp_c'] * 9 / 5 + 32)

          temp_c  temp_f  temp_k
Portland    17.0    62.6  290.15
Berkeley    25.0    77.0  298.15
```

How to handle this in inline documentation?

**Comment by garawalid** [comment]
I have the same issue in #239  & #238. 
I think it's related to Travis CI.

**Comment by shril** [comment]
Thanks @garawalid.

**Comment by rxin** [comment]
I think the way to handle this is to create an actual test case in tests directory, and then mark the doctest with `# doctest: +SKIP`.

In the tests directory you can compare the actual values without caring about the orders in the dictionary.

File: databricks/koalas/frame.py [hunk 1]
```diff
@@ -798,8 +798,62 @@ def to_pandas(self):
...
+
+        Alternatively, the same behavior can be achieved by directly
+        referencing an existing Series or sequence and you can also
+        create multiple columns within the same assign.
+
+        >>> df.assign(temp_f=df['temp_c'] * 9 / 5 + 32,
+        ...           temp_k=df['temp_c'] + 273.15)
+                  temp_c  temp_f  temp_k
+        Portland    17.0    62.6  290.15
+        Berkeley    25.0    77.0  298.15
```
Comment (ueshin) [inline databricks/koalas/frame.py]: Maybe we can do something like:

```py
        >>> assigned = df.assign(temp_f=df['temp_c'] * 9 / 5 + 32,
        ...                      temp_k=df['temp_c'] + 273.15)
        >>> assigned[['temp_c', 'temp_f', 'temp_k']]
```

File: databricks/koalas/frame.py [hunk 2]
```diff
@@ -798,8 +798,62 @@ def to_pandas(self):
...
+
+        Notes
+        -----
+        Assigning multiple columns within the same ``assign`` is possible
+        but you cannot refer to newly created or modified columns. This
+        feature is supported in Pandas for Python 3.6 and later but not in
+        Koalas. In koalas, all items are computed first, and then assigned.
```
Comment (ueshin) [inline databricks/koalas/frame.py]: nit: `In koalas, ` -> `In Koalas, `

File: databricks/koalas/frame.py [hunk 3]
```diff
@@ -798,8 +798,62 @@ def to_pandas(self):
...
+        ...           temp_k=df['temp_c'] + 273.15)
+                  temp_c  temp_f  temp_k
+        Portland    17.0    62.6  290.15
+        Berkeley    25.0    77.0  298.15
+
+        Notes
+        -----
+        Assigning multiple columns within the same ``assign`` is possible
+        but you cannot refer to newly created or modified columns. This
+        feature is supported in Pandas for Python 3.6 and later but not in
```
Comment (ueshin) [inline databricks/koalas/frame.py]: nit: `in Pandas` -> `in pandas`?

File: databricks/koalas/frame.py [hunk 4]
```diff
@@ -798,8 +798,62 @@ def to_pandas(self):
...
+        Returns a new object with all original columns in addition to new ones.
+        Existing columns that are re-assigned will be overwritten.
+
+        Parameters
+        ----------
+        **kwargs : dict of {str: callable or Series}
+            The column names are keywords. If the values are
+            callable, they are computed on the DataFrame and
+            assigned to the new columns. The callable must not
+            change input DataFrame (though koalas doesn't check it).
```
Comment (ueshin) [inline databricks/koalas/frame.py]: nit: `koalas` -> `Koalas`.

**Comment by HyukjinKwon** [comment]
I will address the comments separaetly.

**Pull request merged**